File: Photon Tools/Servers/Game/python/scenes/Map_1/npcs/Guardian.py
from MathematicsHelper import *
from Game.InterestManagement import *

import os
import json

path = os.path.abspath(__file__)
jsonPath = os.path.splitext(path)[0] + ".json"

with open(jsonPath) as data_file:
    data = json.load(data_file)

name = data["Name"]

x = data["Position"]["x"]
y = data["Position"]["y"]

position = Vector2(x, y)
guardian = SceneObject(name, position)

scene.AddSceneObject(guardian)

File: Maple Fighters/Servers/Game/Game/Application/Scene Objects/NPCs/Guardian.cs
using Game.InterestManagement;
using MathematicsHelper;
using ServerApplication.Common.Components;

namespace Game.Application.SceneObjects
{
    public class Guardian : SceneObject
    {
        public Guardian(string name, Vector2 position, Direction direction) 
            : base(IdGenerator.GetId(), name, new TransformDetails(position, Vector2.Zero, direction))
        {
            // Left blank intentionally
        }
    }
}